cluster_tools/cluster_tools/executors/dask.py

```diff
@@ -64,10 +64,13 @@ def submit(  # type: ignore[override]
             ]
             del kwargs["__cfut_options"]
 
-            __fn = partial(
-                MultiprocessingExecutor._execute_and_persist_function,
-                output_pickle_path,
-                __fn,
+            __fn = cast(
+                Callable[_P, _T],
```

Comment:
mypy doesn't understand `partial` yet. But, now it understands enough about `Callable`s to complain.